docs/tools/cli/service/schema-registry-acl.rst

```diff
@@ -0,0 +1,85 @@
+``avn service schema-registry-acl``
+============================================
+
+Here you’ll find the full list of commands for ``avn service schema-registry-acl``.
+
+
+Manage Apache Kafka® Karapace schema registry access control lists
+------------------------------------------------------------------
+
+Commands for managing :doc:`Aiven for Apache Kafka® Karapace schema registry authorization </docs/products/kafka/concepts/schema-registry-authorization>` via ``avn`` commands.
+
+``avn service schema-registry-acl-add``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Adds an Aiven for Apache Kafka® Karapace schema registry ACL entry. The detailed explanation of parameters can be found in the :ref:`dedicated page <karapace_schema_registry_acls>`.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--permission``
+    - The permission type: possible values are ``schema_registry_read`` or ``schema_registry_write``
+  * - ``--resource``
+    - The resource to grant access to, can be in the form of ``Config:`` or ``Subject:<subject>``, accepts * and ? as wildcard characters accepts ``*`` and ``?`` as wildcard characters. More information can be found in the :ref:`dedicated page <karapace_schema_registry_acls>`.
+  * - ``--username``
+    - The username pattern: accepts ``*`` and ``?`` as wildcard characters
+
+**Example:** Add an ACLs for users with username starting with ``userAB`` to write (``schema_registry_write``) subjects having name starting with ``s123`` in the service ``kafka-doc``.
+
+::
+
+  avn service schema-registry-acl-add kafka-doc \
+    --username userAB*                          \
```

Comment:
How about commands here and all over, if this is copy-pasted to local shell and cwd contains file named userABC, this will cause unexpected behavior? How about quoting `'userAB*'`?